Configure

```diff
@@ -1901,7 +1901,7 @@ fi
 touch posthint.sh
 
 : set package name
-package='perl5'
+package='perl7'
```

Comment:
I have not yet found time to dig into rebuilding Configure with all these changes, so I have no idea if anything would be impacted at all. I will need to find time first to experiment and time to backport all changes.